sycl/test-e2e/Matrix/Inputs/common.hpp

```diff
@@ -26,6 +26,7 @@ using bfloat16 = sycl::ext::oneapi::bfloat16;
 // Something like this should be good enough to catch algorithm errors:
 // fabs(ref[i] - val[i])/max(fabs(ref)) < 10e-2
 constexpr float FLOAT_EPSILON = 10e-2;
+constexpr float BFLOAT16_EPSILON = 125e-3; // 0.125
```

Comment:
We don't need new epsilon. `FLOAT_EPSILON` should be ok for bfloat16 cases as well (and it is basically the same order now)